### PR TITLE
ManageMeetings commit

### PR DESCRIPTION
**Pull request by tymu47**
ManageMeetings page commit

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,14 @@
...
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.1",
+    "bootstrap": "^5.0.2",
     "moment": "^2.29.1",
     "react": "^17.0.2",
+    "react-bootstrap": "^1.6.1",
     "react-datepicker": "^4.1.1",
     "react-dom": "^17.0.2",
     "react-icons": "^4.2.0",
+    "react-p5-wrapper": "^2.4.1",
```
Comment (lukegehron) [inline package.json]: do we need this dependancy?

File: src/component/ManageMeetings/ManageMeetings.js [hunk 2]
```diff
@@ -1,14 +1,69 @@
...
+                <li class="nav-item">
+                <a class="nav-link active" href="#">Student</a>
+                </li>
+                <li class="nav-item">
+                <a class="nav-link" href="#">Tutor</a>
+                </li>
+                </ul>
+            </Col>
+            <Col></Col>
+            <Col></Col>
```
Comment (lukegehron) [inline src/component/ManageMeetings/ManageMeetings.js:32]: what do these empty <Col>s do?

Comment (tymu47) [reply src/component/ManageMeetings/ManageMeetings.js:32]: They're for spacing so it divides the grid further, to make those Student/Tutor tabs further from the Search +Schedule Meeting button on the same row. I'm sure there's a better way of doing this, but this seems to work.

File: package.json (hunk 1 above)
Comment (tymu47) [reply package.json]: Not sure, don't know how it got in there. Maybe this was something your Meeting Page needed?

**Pull request merged**